x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/shared_fields/shrink_field.tsx

```diff
@@ -51,22 +66,51 @@ export const ShrinkField: FunctionComponent<Props> = ({ phase }) => {
       }}
       fullWidth
     >
-      <EuiFlexGroup>
-        <EuiFlexItem>
-          <UseField
-            path={path}
-            component={NumericField}
-            componentProps={{
-              fullWidth: false,
-              euiFieldProps: {
-                'data-test-subj': `${phase}-primaryShardCount`,
-                min: 1,
-              },
-            }}
-          />
-        </EuiFlexItem>
-      </EuiFlexGroup>
-      <EuiSpacer />
+      {isUsingShardSize === undefined ? null : (
```

Comment:
This check is needed because the `isUsingShardSize === undefined` on the 1st render and the form would display only `number_of_shards` input first. For policies with shard size configured, that "removes" the shard size value from the form.

Reply:
You can get around by providing the default (`false`). This also remove the need to add support for `null` as children of `<DescribedFormRow >`

```js
export const ShrinkField: FunctionComponent<Props> = ({ phase }) => {
  const globalFields = useGlobalFields();
  const { policy } = useEditPolicyContext();

  const isUsingShardSizePath = `_meta.${phase}.shrink.isUsingShardSize`;
  const [formData] = useFormData({ watch: [isUsingShardSizePath] });
  const isUsingShardSize: boolean = get(formData, isUsingShardSizePath) ?? false;

  ...
```